android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -2746,6 +2746,11 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_TOUCH_SOUND_ENABLED = "soundEnabled";
```

Comment:
We should name it the same (`soundEffectsEnabled`), which would be mapped to `PROPERTY_SOUND_EFFECTS_ENABLED`.